Gruntfile.js

```diff
@@ -447,43 +462,69 @@ module.exports = function( grunt ) {
 					packageManager: "npm"
 				}
 			}
+		},
+		"start-selenium-server": {
+			dev: {
+				options: {
+					downloadUrl: "https://selenium-release.storage.googleapis.com/2.45/" +
+						"selenium-server-standalone-2.45.0.jar",
+					downloadLocation: "node_modules/grunt-selenium-server/",
+					serverOptions: serverOptions,
+					systemProperties: {}
+				}
+			}
+		},
+		"stop-selenium-server": {
+			dev: {}
```

Comment:
Thanks @arschmitz. So, @apsdehal let's remove this change please. More details in 1.2 of https://github.com/jquery/globalize/pull/450/#issuecomment-114141947.